content/go/concepts/math-functions/terms/modf/modf.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: 'Modf()'
+Description: 'Returns the integer and fractional part of a floating-point number.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Modf()`** function in Go is used to find the integer and fractional part of a floating-point number. This function is part of the `math` package, and the package must be imported to use the function in a Go program.
+
+## Syntax
+
+```pseudo
+result := Modf(f float64) (int float64, frac float64)
```

Comment:
```suggestion
result := Modf(f) (int, frac)
```

Reply:
Typically in docs we keep the syntax expression very minimal, and highlight types and other details in the section below that describes any parameters or special cases, etc